delete_duplicates.py

```diff
@@ -0,0 +1,13 @@
+"""
+With a given list [12,24,35,24,88,120,155,88,120,155], write a program to
+print this list after removing all duplicate 
+values with original order reserved.
+"""
+
+
+def original_numbers():
+    repeating_values = [12, 24, 35, 24, 88, 120, 155, 88, 120, 155]
```

Comment:
It's better to use testing data as a parameter in the function instead of hardcoding some data in function.